zipline/finance/performance/period.py

```diff
@@ -402,11 +429,12 @@ def as_account(self):
                     self.ending_cash / (self.ending_cash + self.ending_value))
         account.day_trades_remaining = \
             getattr(self, 'day_trades_remaining', float('inf'))
-        account.leverage = \
-            getattr(self, 'leverage', self._gross_leverage())
-        account.net_leverage = self._net_leverage()
-        account.net_liquidation = \
-            getattr(self, 'net_liquidation', self._net_liquidation_value)
+        account.leverage = getattr(self, 'leverage',
```

Comment:
Yeah, this is actually compat for IB stuff on Quantopian...not how I would have written this, but agreed not in the scope of your change.